airgun/widgets.py

```diff
@@ -2696,29 +2696,29 @@ def read(self):
         return self.column_value.read()
 
 
-class FieldWithEditBtn(Widget):
+class FieldWithEditButton(Widget):
     """A pair of a field and a button that makes the field editable.
     After editing, confirm by checkmark or cancel by X.
     Is present e.g. in PF5 Settings.
     """
 
     ROOT = '//td[2]'
-    textinput = TextInput(locator=".//input[@data-ouia-component-type='PF5/TextInput']")
-    textarea = TextInput(locator=".//textarea")
-    dropdown = FormSelect(locator=".//select[@data-ouia-component-type='PF5/FormSelect']")
-    editBtn = PF5Button(locator=".//button[contains(@data-ouia-component-id, 'edit-row')]")
-    confirmBtn = PF5Button(locator=".//button[@data-ouia-component-id='submit-edit-btn']")
-    cancelBtn = PF5Button(locator=".//button[@data-ouia-component-id='cancel-edit-btn']")
+    text_input = TextInput(locator=".//input[@data-ouia-component-type='PF5/TextInput']")
+    text_area = TextInput(locator=".//textarea")
+    drop_down = FormSelect(locator=".//select[@data-ouia-component-type='PF5/FormSelect']")
+    edit_button = PF5Button(locator=".//button[contains(@data-ouia-component-id, 'edit-row')]")
```

Comment:
@lhellebr PF5OUIAButton can be used also here :D

Reply:
@LadislavVasina1 I don't think so, not easily. See the above comment where I originally forgot to hit the "comment" button :)